### ISSUE TITLE
Program detection in the Hotkey Menu (Request)

### ISSUE THREAD

**Issue by Kalbi1** [Hyrules/WinHue3#184]
## Your setup##

WinHue version : 3.0.2128.0
Bridge api version : newsest
Light model : Hue Bridge 2. Gen + 15 Lights

-------------------------------------------

Hello, I have not found a mistake! But I would find a feature very cool: Is it possible that you are incorporating a feature that tells WinHue which program is currently in the foreground and you can use it to trigger certain scenes? It would be super cool!

Basically I want a simple program recognition in the hotkey menu!

I would be very happy if something would come ... I imagine something like that, I start my movie program and the light is dimmed. Or I start Microsoft Word and it is blatantly white light to concentrate better.

Greetings from Germany
Henrik

**Comment by Hyrules** [comment]
I'll have to take a look on how to do that in C# i'm not sure of the scope of this request and how to implement it if possible. I will see what I can do.

**Comment by Kalbi1** [comment]
OK that sounds great ;)

**Comment by Hyrules** [comment]
Try the latest version it should do what you asked for. When you create an hotkey you have to option to start a program at the same time. You just have to select the exe you want to start.

**Comment by Hyrules** [comment]
Can you let me know if what i have implemented in the latest version is what you are asking ? otherwise I will close the issue and mark it as done.
